Doc/library/cmd.rst

```diff
@@ -217,6 +217,11 @@ Instances of :class:`Cmd` subclasses have some public instance variables:
    :mod:`readline`, on systems that support it, the interpreter will automatically
    support :program:`Emacs`\ -like line editing  and command-history keystrokes.)
 
+.. attribute:: Cmd.hide_undoc
```

Comment:
This should have a `.. versionadded:: 3.8` directive since it's a new attribute.

Reply:
ok, understood, will need a while to figure out how to add new commit to pull request.